### PR TITLE
GG EZ

### PR DESCRIPTION
**Pull request by BenjaminPalko**
Put me in coach

### CODE REVIEW

**Review by BenjaminPalko — COMMENTED** [review]
Im ready lets goooo

**Comment by RenatoSerk** [comment]
Don't add this. This code is absolute trash and you should feel bad.

File: pages/ben.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Ben's page</title>
+    </head>
+    <body>
+
+        <p>Yoooooooo, this is Ben's page.
```
Comment (DeltaF1) [inline pages/ben.html:8]: Add a <br> tag

Comment (BenjaminPalko) [reply pages/ben.html:8]: `<br>` for bruh?

Comment (DeltaF1) [reply pages/ben.html:8]: Tooootally bruh

Comment (RenatoSerk) [reply pages/ben.html:8]: Missing a `"</p>".` Absolute trash man. This guy should just drop out now

**Pull request merged**